stdlib/sql/from.go

```diff
@@ -244,3 +196,40 @@ func (c *SQLIterator) Decode() (flux.Table, error) {
 func (c *SQLIterator) Close() error {
 	return c.db.Close()
 }
+
+func NewMySQLRowReader(r *sql.Rows) (execute.RowReader, error) {
```

Comment:
when you move the above code from execute back to this package,   create 2 files  `mysql.go`  and `postgres.go`  and put these helper functions inside of there.